gguf-py/gguf/vocab.py

```diff
@@ -1,15 +1,23 @@
 from __future__ import annotations
 
+import re
 import logging
 import json
 import os
 from pathlib import Path
-from typing import Any, Callable, Sequence, Mapping, Iterable
+from typing import Any, Callable, Sequence, Mapping, Iterable, Protocol, ClassVar, runtime_checkable
+
+from sentencepiece import SentencePieceProcessor
+
+import gguf
 
 from .gguf_writer import GGUFWriter
 
 logger = logging.getLogger(__name__)
 
+ADDED_TOKENS_FILE = 'added_tokens.json'
+FAST_TOKENIZER_FILE = 'tokenizer.json'
```

Comment:
I'm wondering how to handle these. Having floating references, even as constants, seems questionable. This is fine for now until we figure out how to deal with these.

Reply:
Can we just replace them with explicitly using strings? I think they are used only in one/two places. I just copied them over from `convert.py`, but I missed that `gguf` exports them.